css/style.css

```diff
@@ -242,15 +256,15 @@ i.icon-browser {
   line-height: 50px;
 }
 i.icon-text-editor {
-  background-image: url('/images/icon-text-editor.png');
+  background-image: url("/images/icon-text-editor.png");
 }
 
 i.icon-prompt {
-  background-image: url('/images/icon-prompt.png');
+  background-image: url("/images/icon-prompt.png");
 }
 
 i.icon-browser {
-  background-image: url('/images/icon-browser.png');
+  background-image: url("/images/icon-browser.png");
```

Comment:
Feedback for next time: Keep the changes limited to necessary changes so it's easier to review and we don't have to check code style changes to see if they impact the design.

Reply:
I greatly appreciate your feedback. I'll make sure to keep my changes more focused on my future contributions.